conans/client/tools/files.py

```diff
@@ -363,3 +363,14 @@ def unix2dos(filepath):
 
 def dos2unix(filepath):
     _replace_with_separator(filepath, "\n")
+
+
+def remove_files_by_mask(directory, pattern):
+    removed_names = []
+    for root, _, filenames in os.walk(directory):
+        for filename in filenames:
+            if fnmatch(filename, pattern):
+                fullname = os.path.join(directory, root, filename)
+                os.unlink(fullname)
+                removed_names.append(fullname)
```

Comment:
I think it would be a bit more useful if the ``removed_names`` were relative to the ``directory``, not absolute paths.

Reply:
changed to return relative path